content/courses/mobile/solana-mobile-dapps-with-expo.md

```diff
@@ -168,9 +171,12 @@ JS/TS.
 import { Pedometer } from "expo-sensors";
 ```
 
-Depending on the package, there may be additional setup required. Be sure to
-read the [docs](https://docs.expo.dev/versions/latest/) when working with a new
-package.
+Depending on the package, there may be additional setup required. For example,
+if you're using the `expo-camera` package, you not only need to install the
+package but also configure the appropriate permissions in your `app.json` or
+`AndroidManifest.xml` file for Android and request runtime permissions for
+accessing the camera. Be sure to read the
+[docs](https://docs.expo.dev/versions/latest/) when working with a new package.
```

Comment:
```suggestion
[Expo docs](https://docs.expo.dev/versions/latest/) when working with a new package.
```

Make link specific.

keep in mind most of this is criticism of the old work, not your changes!